herd/tests/instructions/AArch64/A24.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A24
+
+(* Tests ldrsb instruction *)
+{ 0:X0 = 1; 0:X1 = x; int8_t x = 127; }
```

Comment:
For this test to be used by litmus without a warning, it is better to declare `0:X0` to be used as a 64bits register: `int64_t 0:X0 =1`. Alternatively, one may not declare any type (implicit type is `int`, 32 bits) and use the`W` form or register un the code.